WireAuthentication/Sources/WireAuthenticationUI/Views/Root/RootView.swift

```diff
@@ -39,6 +39,7 @@ package struct RootView: View {
         BackgroundView()
             .universalSheet(item: $viewModel.modalDestination) { item in
                 sheetContent(for: item)
+                    .id(item.hashValue)
```

Comment:
suggestion: this seems important, so it would be good to add a comment why it's there.

Reply:
done